src/linux/namespaces.rs

```diff
@@ -286,20 +307,54 @@ bitflags! {
     pub struct MountFlags: libc::c_ulong {
         /// Create a bind mount.
         const BIND = libc::MS_BIND;
-        /// Do not honor set-user-ID and set-group-ID bits or file capabilities when
-        /// executing programs from this filesystem.
-        const NOSUID = libc::MS_NOSUID;
         /// Used in conjuction with [`Self::BIND`] to create a recursive bind mount, and
         /// in conjuction with the propagation type flags to recursively change the
         /// propagation type of all of the mounts in a sub-tree.
         const RECURSIVE = libc::MS_REC;
         /// Make this mount private. Mount and unmount events do not propagate into or
         /// out of this mount.
         const PRIVATE = libc::MS_PRIVATE;
-        /// Mount filesystem read-only.
-        const READONLY = libc::MS_RDONLY;
-        /// Do not allow programs to be executed from this filesystem.
-        const NOEXEC = libc::MS_NOEXEC;
+    }
+}
+
+/// Parameter for the `mount_setattr` syscall.
+#[repr(C)]
+#[derive(Default)]
+struct MountAttr {
+    attr_set: u64,
+    attr_clr: u64,
+    propagation: u64,
+    userns_fd: u64,
+}
+
+bitflags! {
+    /// Flags for the `mount_setattr` syscall.
```

Comment:
Where did the values for these flags come from? How can we be sure they are correct? There is one (`RELATI`) that has a zero value. Can a comment about the reference/source of these flags be added, if there is one?

Reply:
The values come directly from the linux kernel, as referenced by `man 2 mount_setattr`.